index.html

```diff
@@ -1535,7 +1536,7 @@ <h1>Event Partners</h1>
 											<a href="https://github.com/fossasia"><i class="icon fa fa-github"></i></a>
 											<a href="https://www.facebook.com/fossasia/"><i class="icon social_facebook"></i></a>
 											<a href="https://in.linkedin.com/company/fossasia"><i class="icon social_linkedin"></i></a>
-											<!-- <a href="#"><i class="icon social_linkedin"></i></a> -->
+											<a href="https://in.linkedin.com/company/fossasia"><i class="icon social_linkedin"></i></a>
```

Comment:
This line seems to be same as [L-1538](https://github.com/fossasia/2018.fossasia.org/pull/139/files#diff-eacf331f0ffc35d4b482f1d15a887d3bR1538)

Reply:
There were two instances.

Reply:
PFA the screenshot. It's repeated. The screenshot was taken from the test link provided by you above.

![screen shot 2018-01-06 at 7 36 44 pm](https://user-images.githubusercontent.com/9530293/34640466-0aac2dda-f319-11e7-9679-0474321989e3.png)

Reply:
:open_mouth: 

Thanks!